serialisation/src/test/java/org/ehrbase/serialisation/flatencoding/FlatJsonTest.java

```diff
@@ -402,12 +389,6 @@ private void check(
 
     softAssertions.assertThat(unmarshal).isNotNull();
 
-    try {
-      new Validator(templateProvider.find(templateId).get()).check(unmarshal);
-    } catch (Exception e) {
-      softAssertions.fail(e.getMessage());
-    }
```

Comment:
Please add the new compostionValidate where ever you removed the old one.

Reply:
I cannot use CompositionValidator here because validation module now depends on serialization and we will have a circular dependency issue.
![image](https://user-images.githubusercontent.com/28752830/151578702-43015c87-afdf-4e5c-9c9e-9ee9bcdd4ddb.png)

But to keep these test cases, I have created the [FlatCompositionValidatorTest](https://github.com/ehrbase/openEHR_SDK/blob/CDR-45_web_template_validation/validation/src/test/java/org/ehrbase/validation/FlatCompositionValidatorTest.java) that validates the same compositions.